### ISSUE TITLE
A few missed characters

### ISSUE THREAD

**Issue by FISHMANPET** [gbuktenica/Unicode-Substitutions#4]
Somehow not all of these hyphens are getting picked up in a Powershell script:
```powershell
    "–"
    $temp = Get–Date
    $temp.AddDays(―1)
    $temp = 1
    $temp ――
    12 ―eq 12
    Get-Date –Date
```
The first two are caught, and the last one is, but lines 3, 5, and 6 are not being detected

**Comment by gbuktenica** [comment]
Unicode Character U+2015 needs a default rule to replace it to U+002D

**Comment by gbuktenica** [comment]
Resolved in v2.2.1
